blazor/treegrid/state-management.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: State management in Blazor TreeGrid Component | Syncfusion
+description: Learn here all about State management in Syncfusion Blazor TreeGrid component and more.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# State management in Blazor TreeGrid Component
+
+Tree Grid Component retains its state using local storage on browser reload. Also it allows users to save and load tree grid state manually. The tree grid will use user-provided state to render instead of its properties provided declaratively.
```

Comment:
Tree Grid Component retains its state using local storage on browser reload. Also, it allows users to save and load tree grid state manually. The tree grid will use a user-provided state to render instead of its properties provided declaratively.

Reply:
made the changes